pkg/kubernetes/scripts/projects.js

```diff
@@ -940,10 +977,13 @@
 
                 return methods.check(request, { "metadata.name": "#project-new-name" })
                     .then(function() {
-                        return methods.create(request)
-                            .then(function() {
-                                return projectData.shareImages(name, fields.access === "shared");
-                            });
+                        return methods.create(request);
+                    })
+                    .then(function(){
+                        return projectData.shareImages(name, fields.access === "shared");
+                    })
+                    .then(function(){
+                        return projectData.makeRegistryAdmin(name, dialogData.currUser);
```

Comment:
As mentioned in irc. 
'admin' user is created by default template.'admin'  user should not be used in registry UI since we have registry-admin for Admin related roles.
makeRegistryAdmin will enforce this even when default template is used.
There is ofcourse no way to add/remove 'admin' role from registry UI.Which makes this safe.